lua/astrocommunity/test/nvim-coverage/nvim-coverage.lua

```diff
@@ -0,0 +1,8 @@
+return {
+  "andythigpen/nvim-coverage",
+  lazy = false,
```

Comment:
```suggestion
  event = "User AstroFile",
```

or check for a filetypes with `ft = { "..." }` or load the plugin when [running a command](https://github.com/andythigpen/nvim-coverage/blob/e542b6afbc7d92acd6502e15124ca2f376964b76/lua/coverage/init.lua#L18). Just disabling lazy-loading has performance implications for startup when the plugin is not actually needed.